homeassistant/components/google_assistant_sdk/__init__.py

```diff
@@ -101,19 +108,30 @@ async def async_unload_entry(hass: HomeAssistant, entry: ConfigEntry) -> bool:
 async def async_setup_service(hass: HomeAssistant) -> None:
     """Add the services for Google Assistant SDK."""
 
-    async def send_text_command(call: ServiceCall) -> None:
+    async def send_text_command(call: ServiceCall) -> ServiceResponse:
```

Comment:
```suggestion
    async def send_text_command(call: ServiceCall) -> ServiceResponse | None:
```

Reply:
ServiceResponse type already includes None
https://github.com/home-assistant/core/blob/23d5fb962211f95cce9dc34458fa321752399b82/homeassistant/core.py#L134

A similar example that doesn't specify None: https://github.com/home-assistant/core/blob/23d5fb962211f95cce9dc34458fa321752399b82/homeassistant/components/conversation/__init__.py#L159